lms/envs/common.py

```diff
@@ -848,6 +848,15 @@
 ##### Feedback submission mechanism #####
 FEEDBACK_SUBMISSION_EMAIL = None
 
+#### Help Modal Links ####
+if FEATURES.get('ENABLE_FEEDBACK_SUBMISSION'):
```

Comment:
Ah, I see. I guess we could go either way. My inclination would still be to assign it unconditionally (it's simpler logic and less likely to break), though now I see what you mean about this var only being accessed if the feature is enabled.